File: week-04/day-01/sharpie.py
''' Create Sharpie class
We should know about each sharpie their color (which should be a string), width (which will be a floating point number), ink_amount (another floating point number)
When creating one, we need to specify the color and the width
Every sharpie is created with a default 100 as ink_amount
We can use() the sharpie objects
which decreases inkAmount '''

class Sharpie(object):
    def __init__(self):
        self.color = str(color)
        self.width = float(width)
        self.ink_amount = float(100)

    def use(self):
        self.ink_amount -= 9.5

yellow = Sharpie()
black = Sharpie()
blue = Sharpie()


    

File: week-08/day-01/sharpie.js
function Sharpie (color, width, inkAmount) {
    this.color = color;
    this.width = width;
    this.inkAmount = 100;
    this.use = function() {
        this.inkAmount -= this.width;
        if (this.inkAmount < 0) {
            this.inkAmount = 0;
        }
    }
}

let newSharpie = new Sharpie ('green', 40);
while (newSharpie.inkAmount > 0) {
    newSharpie.use();
    console.log(newSharpie.inkAmount);
}